tests/playwright/specs/wikipediaHomePage.js

```diff
@@ -0,0 +1,25 @@
+const { chromium } = require( 'playwright' );
+
+describe( 'Wikipedia Home Page', () => {
+	let response, browser, page;
+
+	beforeAll( async () => {
+		browser = await chromium.launch();
+		page = await browser.newPage();
+		response = await page.goto( 'https://www.wikipedia.org/' );
+	} );
+
+	afterAll( async () => {
+		await browser.close();
+	} );
+
+	it( 'test if page loads successfully', async () => {
+		expect( response.status() ).toBe( 200 );
+	} );
+
+	it( 'should check if page sub title is correct', async () => {
```

Comment:
Same as above

Reply:
Make a single test which checks only the subtitle as if the subtitle loads it implies that the page has loaded correctly